### PR TITLE
Documentation about configuring multiple cameras

### PR DESCRIPTION
**Pull request by derlaft**
[Current documentation says to check `cameras.md` for info about configuring multiple cameras.](https://docs.fluidd.xyz/faq#id-like-to-setup-multiple-cameras-how)

However, there isn't much information about it. This commit tries to address it, documenting the process of configuring multiple mjpeg_streamer instances.

### CODE REVIEW

**Review by sjqlwy — APPROVED** [review]
nice work, details can be found on https://docs.mainsail.xyz/quicktips/multicam

**Comment by derlaft** [comment]
Huh, that's a much better version of what I was trying to create, really regret not finding that myself :) Probably makes no sense to use this version since a superior one already exists

**Comment by eduncan911** [comment]
> Huh, that's a much better version of what I was trying to create, really regret not finding that myself :) Probably makes no sense to use this version since a superior one already exists

@sjqlwy @derlaft, should we link to Mainsail's document?

@pedrolamas, is there a precedence of linking to Mainsail for documents and config settings?  If not, then perhaps we do keep this file and add a bunch of info to it?

**Comment by derlaft** [comment]
> is there a precedence of linking to Mainsail for documents and config settings? If not, then perhaps we do keep this file and add a bunch of info to it?

Both projects are GPL, you can just copy mainsail docs into here.

**Comment by eduncan911** [comment]
> > is there a precedence of linking to Mainsail for documents and config settings? If not, then perhaps we do keep this file and add a bunch of info to it?
> 
> Both projects are GPL, you can just copy mainsail docs into here.

Lol, true... But that just feels, wrong.  Heh.

**Comment by pedrolamas** [comment]
> @pedrolamas, is there a precedence of linking to Mainsail for documents and config settings? If not, then perhaps we do keep this file and add a bunch of info to it?

No idea...

Having said that, I don't think it is a big issue to copy and adapt Mainsail docs to here.

In the end of the day, we are all working in benefit of the Klipper user-community and we should have as much alignment on documentation of shared features/functionality as possible.

In any case, I think it best to just drop a message to Mainsail mods just to make they do agree with this view.

**Comment by eduncan911** [comment]
@derlaft could you reach out to some Mailsail devs on their chat/discord/etc and see if they are fine with replicating some documentation between the projects?

**Review by pedrolamas — DISMISSED** [review]
LGTM

File: docs/features/cameras.md [hunk 1]
```diff
@@ -44,3 +44,27 @@ Currently supported types are;
 Head on over to the UI Settings page to define your cameras.
 
 ![screenshot](/assets/images/camera_settings.png)
+
+# Setting up multiple cameras on FluiddPi
+
+Up to four cameras are supported by the stock configuration.
+In order to make them work, you need to
+
+1. Copy `webcam.txt` configuration file into a new one
```
Comment (matmen) [inline docs/features/cameras.md:53]:
```suggestion
1. Copy the `webcam.txt` configuration file into a new one 
```

File: docs/features/cameras.md [hunk 2]
```diff
@@ -44,3 +44,27 @@ Currently supported types are;
...
 
 ![screenshot](/assets/images/camera_settings.png)
+
+# Setting up multiple cameras on FluiddPi
+
+Up to four cameras are supported by the stock configuration.
+In order to make them work, you need to
+
+1. Copy `webcam.txt` configuration file into a new one 
+   matching `webcam*.txt` wildcard.
```
Comment (matmen) [inline docs/features/cameras.md:54]:
```suggestion
   matching the `webcam*.txt` wildcard.
```

File: docs/features/cameras.md [hunk 3]
```diff
@@ -44,3 +44,27 @@ Currently supported types are;
...
+   Make sure each configuration file to a different camera.
```
Comment (matmen) [inline docs/features/cameras.md:55]:
```suggestion
   Make sure each configuration file points to a different camera.
```

File: docs/features/cameras.md [hunk 4]
```diff
@@ -44,3 +44,27 @@ Currently supported types are;
...
+   For example, one configuration file might have `camera="usb"` and
+   the other one `camera="auto"`.
+   Make any other adjustments if needed.
+3. Make sure each configuration file has a different port settings.
+   By default `mjpg_streamer` will listen on port `8080`.
+   Up to four cameras are supported by the default nginx configuration,
+   from port `8080` to `8083`. Configure it by setting up
+   `camera_http_options` parameter.
```
Comment (matmen) [inline docs/features/cameras.md:63]:
```suggestion
   the `camera_http_options` parameter.  
```

File: docs/features/cameras.md [hunk 5]
```diff
@@ -44,3 +44,27 @@ Currently supported types are;
...
+   Example: `camera_http_options="-n -p 8081"`
+4. Restart webcamd via SSH: `sudo systemctl restart webcamd`.
```
Comment (matmen) [inline docs/features/cameras.md:65]: webcamd can now also be restarted using the UI:
```suggestion
4. Restart webcamd either via the tools drawer in the top right of the UI
   or via SSH: `sudo systemctl restart webcamd`.
```

**Review by matmen — CHANGES_REQUESTED** [review]

**Review by pedrolamas — APPROVED** [review]

**Review by matmen — APPROVED** [review]

**Pull request merged**